guide/sending-delivery-schedule/sending-my-first-order.md

```diff
@@ -0,0 +1,113 @@
+# Option A: Sending a Delivery Schedule per order line
+
+This page explains how to request deliveries for order lines by means of a Delivery Schedule. A Delivery Schedule can contain 0, 1 or multiple requested deliveries for a single order line.
+
+### Sending the order
+
+1. Set the URL to `https://api.accp.tradecloud1.com/v2/api-connector/order`
+2. Set the HTTP Method to `POST`
+3. Provide a **Basic Authentication** header, which contains the [username and password](getting-started.md#2-getting-an-integration-account) of the integration account.
+4. Provide the JSON below as the request body. Make sure you replace `{{supplierAccountNumber}}` with the supplier account number you have [configured](#configure-the-supplier-account-number) for a test supplier.
+
+
+{% tabs %}
+{% tab title="JSON" %}
+
+Set the request body MIME-type to `application/json`.
+
+```json
+{
+  "order": {
+    "supplierAccountNumber": "{{supplierAccountNumber}}",
+    "purchaseOrderNumber": "PO123456789",
+    "destination": {
+      "names": [
+        "My Company Warehouse",
+        "Dock 12"
+      ],
+      "countryCodeIso2": "NL"
+    }
+  },
+  "lines": [
+    {
+      "position": "0001",
+      "item": {
+        "name": "Round tube 60x45",
+        "purchaseUnitOfMeasureIso": "PCE"
+      },
+      "prices": {
+        "netPrice": {
+          "priceInTransactionCurrency": {
+            "value": 1234.56,
+            "currencyIso": "EUR"
+          }
+        },
+        "priceUnitOfMeasureIso": "PCE",
+        "priceUnitQuantity": 100
+      },
+      "deliverySchedule": [
```

Comment:
move `deliverySchedule` above prices, so it is more visible

Reply:
Updated

Reply:
Its still below prices

Reply:
Good catch. Updated.